internal/worker/cmd_meta.go

```diff
@@ -107,6 +107,12 @@ const (
 	CmdCMSInitByProb = "CMS.INITBYPROB"
 	CmdCMSMerge      = "CMS.MERGE"
 	CmdCMSIncrBy     = "CMS.INCRBY"
+	CmdHset        = "HSET"
+	CmdHget        = "HGET"
+	CmdHsetnx      = "HSETNX"
+	CmdHdel        = "HDEL"
+	CmdHmset       = "HMSET"
+	CmdHmget       = "HMGET"
```

Comment:
```suggestion
	CmdHSet        = "HSET"
	CmdHGet        = "HGET"
	CmdHSetNx      = "HSETNX"
	CmdHDel        = "HDEL"
	CmdHMSet       = "HMSET"
	CmdHMGet       = "HMGET"
```

also, please use prettier to keep the space consistent